<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test2Dialog</class>
 <widget class="QDialog" name="test2Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>低温泵</string>
  </property>
  <property name="windowIcon">
   <iconset resource="errordetection.qrc">
    <normaloff>:/images/images/timg.jpeg</normaloff>:/images/images/timg.jpeg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>50</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>低温泵1级冷头温度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test21">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>54</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QWidget" name="widget_5" native="true">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>23</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>低温泵3#胀板温度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test25">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>54</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test25">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="widget_3" native="true">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>低温泵1#胀板温度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test23">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>54</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test23">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>50</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>低温泵2级冷头温度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test22">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>54</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test22">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_9" native="true">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>60</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>  低温泵供电状态</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test26">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>54</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test26">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget_10" native="true">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>59</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>低温泵1级加热器运行状态</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test27">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>54</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test27">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QWidget" name="widget_12" native="true">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>60</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>低温泵状态</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test29">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>54</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test29">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QWidget" name="widget_13" native="true">
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>23</x>
        <y>60</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>低温泵运行电流</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test210">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>54</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test210">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QWidget" name="widget_11" native="true">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>60</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>低温泵2级加热器运行状态</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test28">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>54</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test28">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QWidget" name="widget_4" native="true">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>50</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>低温泵2#胀版温度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test24">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>54</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test24">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="errordetection.qrc"/>
 </resources>
 <connections/>
</ui>
